### PR TITLE
add branch to git_info

### PR DESCRIPTION
**Pull request by Salzig**
add current_branch to git_info

### CODE REVIEW

**Comment by mroth** [comment]
@Salzig This seems to be timing out the test suite on travis -- any idea what's going on?

**Comment by Salzig** [comment]
@mroth no, not sure. But fixed another problem. Would be nice if we could merge #253 to fix rubocop problems.

**Comment by Salzig** [comment]
rebased onto master

**Comment by Salzig** [comment]
@mroth looks good now :)

File: lib/lolcommits/git_info.rb [hunk 1]
```diff
@@ -3,50 +3,77 @@ module Lolcommits
...
       debug "GitInfo: \t#{author_name}" if author_name
       debug "GitInfo: \t#{author_email}" if author_email
     end
 
+    def branch
+      @branch ||= repository.current_branch
+    end
+
+    def message
+      @message ||= "test" #last_commit.message.split("\n").first
```
Comment (matthutchinson) [inline lib/lolcommits/git_info.rb]: shouldn't this comment be used, i.e.

`@message ||= last_commit.message.split("\n").first`

**Comment by matthutchinson** [comment]
:+1: Looks good to me and a great refactor of the initialize method, thanks for cleaning that up.  Just one comment on the `def message` method

File: lib/lolcommits/git_info.rb (hunk 1 above)
Comment (Salzig) [inline lib/lolcommits/git_info.rb]: oh, sure.

**Comment by matthutchinson** [comment]
Thanks again, merging :+1:

**Pull request merged**